editor/window_wrapper.cpp

```diff
@@ -0,0 +1,442 @@
+	for (int i = 0; i < DisplayServer::get_singleton()->get_screen_count(); i++) {
+		Button *button = memnew(Button);
+
+		Size2 screen_size = Size2(DisplayServer::get_singleton()->screen_get_size(i));
+		Size2 button_size = Size2(height * (screen_size.x / screen_size.y), height);
+		button->set_custom_minimum_size(button_size);
+		screen_list->add_child(button);
+
+		button->set_text(itos(i));
+		button->set_text_alignment(HORIZONTAL_ALIGNMENT_CENTER);
+		button->set_tooltip_text(vformat(TTR("Make this panel floating in the screen %d"), i));
+
+		if (i == current_screen) {
+			Color accent_color = get_theme_color("accent_color", "Editor");
+			button->add_theme_color_override("font_color", accent_color);
+		}
+
+		button->connect("pressed", callable_mp(this, &ScreenSelect::_emit_screen_signal).bind(i));
+		button->connect("pressed", callable_mp(static_cast<BaseButton *>(this), &ScreenSelect::set_pressed).bind(false));
+	}
+}
+
+void ScreenSelect::_emit_screen_signal(int p_screen_idx) {
+	emit_signal("request_open_in_screen", p_screen_idx);
+}
+
+void ScreenSelect::_bind_methods() {
+	ADD_SIGNAL(MethodInfo("request_open_in_screen", PropertyInfo(Variant::INT, "screen")));
+}
+
+void ScreenSelect::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_READY: {
+			connect("about_to_popup", callable_mp(this, &ScreenSelect::_build_advanced_menu));
+			connect("gui_input", callable_mp(this, &ScreenSelect::_handle_mouse_shortcut));
+		} break;
+		case NOTIFICATION_THEME_CHANGED: {
+			set_icon(EditorNode::get_singleton()->get_gui_base()->get_theme_icon("MakeFloating", "EditorIcons"));
+		} break;
+	}
+}
+
+void ScreenSelect::_handle_mouse_shortcut(const Ref<InputEvent> &p_event) {
+	const Ref<InputEventMouseButton> mouse_button = p_event;
+	if (mouse_button.is_valid()) {
+		if (mouse_button->is_pressed() && mouse_button->get_button_index() == MouseButton::LEFT) {
+			_emit_screen_signal(get_window()->get_current_screen());
+			accept_event();
+		}
+	}
+}
+
+ScreenSelect::ScreenSelect() {
+	set_text(TTR("Make Floating"));
+	set_tooltip_text(TTR("Make this panel floating.\nRight click to open the screen selector."));
+	set_button_mask(MouseButtonMask::RIGHT);
+
+	// Create the popup.
+	const real_t popup_height = real_t(get_theme_font_size("font_size")) * 2.0;
+	const Size2 borders = Size2(4, 4) * EDSCALE;
+
+	get_popup()->set_min_size(Size2(0, popup_height * 3));
+
+	MarginContainer *popup_root = memnew(MarginContainer);
+	popup_root->add_theme_constant_override("margin_right", borders.width);
+	popup_root->add_theme_constant_override("margin_top", borders.height);
+	popup_root->add_theme_constant_override("margin_left", borders.width);
+	popup_root->add_theme_constant_override("margin_bottom", borders.height);
+	get_popup()->add_child(popup_root);
+
+	VBoxContainer *vb = memnew(VBoxContainer);
+	vb->set_alignment(BoxContainer::ALIGNMENT_CENTER);
+	popup_root->add_child(vb);
+
+	Label *description = memnew(Label(TTR("Select screen")));
```

Comment:
```suggestion
	Label *description = memnew(Label(TTR("Select Screen")));
```
We seem to be using this casing everywhere.